scripts/cybershake/install_cybershake.py

```diff
@@ -45,6 +46,13 @@ def load_args(logger):
         default="/nesi/project/nesi00213/StationInfo/non_uniform_whole_nz_with_real_stations-hh400_v20p3_land.ll",
         help="The path to the station info file path.",
     )
+    parser.add_argument(
+        "--components",
+        choices=list(Components.iterate_str_values()),
+        nargs="+",
+        default=None,
```

Comment:
Where's the default stored? Did we also need to update the 20.* templates?

Reply:
james has already updated the templates with his previous PR.

Reply:
the question is do we want to have a deafult in the gmsim template and assume we will always have it there, or do we want a defautl value in the qcore libs.